addons/crm/models/crm_lead.py

```diff
@@ -635,7 +635,7 @@ def copy(self, default=None):
         # Do not assign to an archived user
         if not self.user_id.active:
             default['user_id'] = False
-        return super(Lead, self.with_context(context)).copy(default=default)
+        return super(Lead, self.with_context(context).sudo()).copy(default=default)
```

Comment:
I really don't see the link between the commit message and the fix.
Adding a random sudo() seems suspicious.